doc/2/core-classes/profile/next/snippets/fromsize.dart

```diff
@@ -0,0 +1,20 @@
+for (var i = 1; i <= 5; i++) {
+  await kuzzle.security.createProfile(
+    'myProfile-${i}',
+    [
+      {
+        'roleId': 'default'
+      }
+    ]
+  );
+}
+
+var res = await kuzzle.security.searchProfiles(query:{
+  'roles': ['default']}, from: 1, size: 1);
```

Comment:
(nitpicking) more readable line wrapping:

```suggestion
var res = await kuzzle.security.searchProfiles(
  query:{ roles': [ 'default' ] }, 
  from: 1, 
  size: 1);
```